pinot-query-planner/src/main/java/org/apache/pinot/query/catalog/PinotCatalog.java

```diff
@@ -127,4 +128,20 @@ public boolean isMutable() {
   public Schema snapshot(SchemaVersion version) {
     return this;
   }
+
+  private static boolean isNullEnabled(TableCache tableCache, String rawTableName) {
+    TableConfig tableConfig = tableCache.getTableConfig(
```

Comment:
We need to get TableConfig for at least one type in a bunch of places in the code. Can we add a util method somewhere?

Reply:
good idea but let's follow up?